perf_dashboard/dockerfile

```diff
@@ -1,19 +1,26 @@
+############################################################
+# Dockerfile to run a Django-based web application
+# Based on an Ubuntu Image
+############################################################
+
+# Set the base image to use to Ubuntu
 FROM ubuntu:latest
 
+# Set the file maintainer
+MAINTAINER Carolyn Hu
```

Comment:
don't add this, i think its deprecated by docker anyways

Reply:
oh, ok...